src/App.test.js

```diff
@@ -1,15 +1,15 @@
 import React from 'react';
-import { render } from '@testing-library/react';
-import { Provider } from 'react-redux';
-import { store } from './app/store';
+import { MemoryRouter } from 'react-router-dom';
+import { render, cleanup } from '@testing-library/react';
```

Comment:
- [ ] The app should have unit tests for several components of the app and some pure functions in Redux code, good work on creating some tests however kindly consider adding tests for your component, not just App.js and redux, this way your app will be well tested